src/components/z-book-card/styles.css

```diff
@@ -70,8 +75,13 @@
   content: "";
 }
 
+:host > article div.content {
+  margin: var(--space-unit) 0;
+  border-radius: var(--border-radius);
+}
+
 :host div.title {
-  color: var(--color-text01);
+  color: var(--color-text02);
   font-size: var(--font-size-4);
```

Comment:
Faccio caso ora che, volendo, abbiamo i token typography per i testi. Ad esempio, in questo caso avremmo `body-2-sb` al posto di `font-size`, `font-weight` e `line-height`. Casi analoghi per gli altri testi della card.

Reply:
Con lo shadow dom temo che non si possa fare.. Perché il componente non vede le classi